### DISCUSSION TITLE
Vuex and Pinia on the same project

### DISCUSSION THREAD

**Discussion by george-dragnea** [quasarframework/quasar#15470] [General - Components / Directives / etc]
I currently use vuex and want to migrate to pinia, but I don't want to migrate the entire system once.
How can I add pinia alongside vuex?

I have added pinia with yarn, create a boot file pinia.js (which contains the code from a stores/index.js from a quasar with pinia project) and add in this boot file app.use(pinia).
It seems like it's working but; I haven't managed to make it work in axios.js and in router/index.js.

**Comment by metalsadman** [comment]
pinia creation needs to be inside the boot exported default function, ie. `export default() => { const someStore = useStore() }`.
